pkg/cache/cache.go

```diff
@@ -47,7 +47,7 @@ func (cache *SchedulerCache) GetNodeinfos() []*NodeInfo {
 
 // build cache when initializing
 func (cache *SchedulerCache) BuildCache() error {
-	log.Println("debug: begin to build scheduler cache")
+	log.V(100).Info("debug: begin to build scheduler cache")
```

Comment:
Why setting log level to 100? Could you set a smaller level such as 5?

Reply:
Sure.